src/bucket/BucketTests.cpp

```diff
@@ -835,6 +835,64 @@ TEST_CASE("bucket persistence over app restart", "[bucket][bucketpersist]")
     }
 }
 
+TEST_CASE("bucket count", "[bucket][count]")
```

Comment:
I wonder if we should make `kNumLevels` a config (we can still hardcode that the app is never started with something other than 10 outside of the `--test` code path in main), such that we can test the behavior up to kNumLevels. Right now the test cases cannot spill up to 10, and therefore we cannot observe the "steady state" behavior from test cases.

Reply:
In order to create a proper test case for issue #1300, we will need to be able to get entries into the bottom level of the BucketList since it is the only place where a merge can cause the elimination of a DEADENTRY and corresponding LIVEENTRY.